index.go

```diff
@@ -309,14 +309,6 @@ func NewBuilder(path string, mapping mapping.IndexMapping, config map[string]int
 	return newBuilder(path, mapping, config)
 }
 
-// IndexCopyable is an index which supports an online copy operation
-// of the index.
-type IndexCopyable interface {
-	// CopyTo creates a fully functional copy of the index at the
-	// specified destination directory implementation.
-	CopyTo(d index.Directory) error
-}
```

Comment:
Bring this and the supporting changes back to here - let's not use https://github.com/blevesearch/bleve_index_api/pull/48.